code/modules/modular_computers/hardware/battery_module.dm

```diff
@@ -2,58 +2,90 @@
 // have tremendeous capacity in comparsion. Higher tier cells would provide your device with nearly infinite battery life, which is something i want to avoid.
 /obj/item/weapon/computer_hardware/battery_module
 	name = "standard battery"
-	desc = "A standard power cell, commonly seen in high-end portable microcomputers or low-end laptops. It's rating is 750."
+	desc = "A standard power cell, commonly seen in high-end portable microcomputers or low-end laptops. It's rating is 75 Wh."
 	icon_state = "battery_normal"
 	critical = 1
-
-	var/battery_rating = 750
-	var/obj/item/weapon/cell/large/battery = null
+	malfunction_probability = 1
+	origin_tech = list(TECH_POWER = 1, TECH_ENGINEERING = 1)
+	var/battery_rating = 75
```

Comment:
Why are all of these batteries 10% of the rating of bay ones?

Reply:
they are equal